CONTRIBUTING.md

```diff
@@ -430,14 +430,14 @@ gitGraph
 
 Here's an overview of the different branches:
 
-| branch | `master` | `staging-next` | `staging` |
-| --- | --- | --- | --- |
-| Used for development | ✔️ | ❌ | ✔️ |
-| Built by Hydra | ✔️ | ✔️ | ❌ |
-| [Mass rebuilds][mass-rebuild] | ❌ | ⚠️  Only to fix Hydra builds | ✔️ |
-| Critical security fixes | ✔️ for non-mass-rebuilds | ✔️ for mass-rebuilds | ❌ |
-| Automatically merged into | `staging-next` | `staging` | - |
-| Manually merged into | - | `master` | `staging-next` |
+| branch | `master` | `staging-next` | `staging` | [`batch-with-kernel-updates`](#test-driver-rebuild) |
+| --- | --- | --- | --- | --- |
+| Used for development | ✔️ | ❌ | ✔️ | ✔️ |
+| Built by Hydra | ✔️ | ✔️ | ❌ | ❌ |
+| [Mass rebuilds][mass-rebuild] | ❌ | ⚠️  Only to fix Hydra builds | ✔️ | ❌ (except kernel changes) |
```

Comment:
Not quite accurate, because we can take any mass rebuild that's mostly only NixOS tests, right?

Reply:
That doesn't capture that we can also take e.g. ruff.

Reply:
It's unfortunate we have so little room in the table.  "Except changes that cause no more rebuilds than kernel updates", or something, but that's probably too long.

Reply:
Another alternative might be to add a footnote if spacing in a table cell is an issue.